src/static/js/modules/jquery/cf_inputSplit.js

```diff
@@ -4,7 +4,8 @@
 
 'use strict';
 
-require( 'jquery' );
+var $ = require( 'jquery' );
+var jQuery = $;
```

Comment:
ESLint thinks `$` is undefined if these aren't included. All of these JS modules need to be attended to individually, tests added, etc. so I didn't think it worth the time to investigate why on this module it had an issue with `$` not being defined. Browserify will resolve the multiple requires for jquery across the codebase anyway. We may end up with an extra `var` in the minified code, but I don't anticipate this being a permanent addition. Added a code comment in https://github.com/cfpb/cfgov-refresh/commit/4c8bac521291627b6094c16bec1307fb0cc8f195